web/static.go

```diff
@@ -90,6 +93,37 @@ func staticFilesHandler(handler http.Handler) http.Handler {
 	})
 }
 
+func pluginPublicFiles(c *Context, w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Cache-Control", "max-age=31556926, public")
```

Comment:
You and me know where this number came up from. But mind adding a comment for it.